hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientExceptionFactory.java

```diff
@@ -0,0 +1,521 @@
+/*
```

Comment:
Can we just list the error codes in a more generic way in a template file and have this class generated? We may need similar error handlers in every language. If we provide a way to generate, it may help the developers.